include/godot_cpp/classes/wrapped.hpp

```diff
@@ -52,7 +54,15 @@ class Wrapped {
 	friend class ClassDB;
 	friend void postinitialize_handler(Wrapped *);
 
+	friend _ALWAYS_INLINE_ void internal::_set_construct_info(const StringName *p_extension_class_name, const GDExtensionInstanceBindingCallbacks *p_binding_callbacks);
+
+	thread_local static const StringName *_constructing_extension_class_name;
+	thread_local static const GDExtensionInstanceBindingCallbacks *_constructing_class_binding_callbacks;
+
 protected:
+	bool has_object_instance_binding() const;
```

Comment:
https://github.com/godotengine/godot-cpp/blob/340dde31a28222ed09a68176b5c658e531fd73d9/include/godot_cpp/classes/wrapped.hpp#L107-L108

Because of this comment, so I think it should be check in wrapped.

Reply:
Hm. Since this only for a test in godot-cpp, I think it should be fine to access `_owner`. I personally think that's better than adding a new API to `Wrapped` for which there is no use-case except this test.

Reply:
OK, I moved it from `Wrapped` to `Example`.